### PR TITLE
Apply upcoming core styling to file upload

### PR DESCRIPTION
**Pull request by timja**
see https://github.com/jenkinsci/jenkins/pull/7452

(untested but should be fine from what I can see in the PR as most of the inputs didn't have classes before)

### CODE REVIEW

**Review by felfert — APPROVED** [review]
I just have one question about the TODO comment:

Can you be more specific about what exactly will have to be changed into an f:file
The input tag or the wrapping f:entry?

File: src/main/resources/jenkins/plugins/jclouds/credentials/JCloudsUserWithKey/credentials.jelly [hunk 1]
```diff
@@ -17,7 +17,8 @@ limitations under the License.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
   <f:entry title="${%Import Google JSON key}" field="jsonFile">
-    <input type="file" jsonAware="yes" accept="application/json" name="jsonFile" />
+    <!-- TODO switch to f:file after baseline includes https://github.com/jenkinsci/jenkins/pull/7452 -->
```
Comment (timja) [inline src/main/resources/jenkins/plugins/jclouds/credentials/JCloudsUserWithKey/credentials.jelly:20]:
```suggestion
    <!-- TODO 2.385+ switch to f:file, see https://weekly.ci.jenkins.io/design-library/File/ -->
```

**Comment by timja** [comment]
> The input tag or the wrapping f:entry?

The input tag, is that TODO clearer?

**Comment by felfert** [comment]
Yes. perfect. Thank you

**Pull request merged**